### ISSUE TITLE
Uncertainty maps for facade parsing training

### ISSUE THREAD

**Issue by manuelaceron** [wohaiyo/PFLNet#1]
Hello, 
Great work!

I'm trying to replicate your results, and I have couple questions:

1. "train_facade_parsing.py" reads directly the image, ground truth and uncertainty map, how do you generate this uncertainty map? Do you have some inference script to generate it for the whole training/validation set using the model trained in "train_uncertainty.py"?

3. In the paper you mention 5-fold cross validation is used, however in the training procedure I don't find this is done, am I missing something?

Thanks!!

**Comment by wohaiyo** [comment]
We have two steps to train the proposed model.   First, we train the Bayesian CNN model, we use the Bayesian model to generate uncertainty maps using in script "train_facade_parsing.py".   The uncertainty map inference script is not in the repository.   It is easy to write a script to realize.

You can write a script to implement 5-fold cross-validation.

**Comment by Abbsalehi** [comment]
Hi, nice work! I'm also replicating your research and have a question. Is there a specific reason why the Bayesian CNN model hasn't been validated to prevent overfitting? I noticed that the default training steps are set to 6000, which might result in the model overfitting. What are your thoughts on this? Thanks!
